### DISCUSSION TITLE
Emojis in command names

### DISCUSSION THREAD

**Discussion by Commandcracker** [Textualize/textual#4580] [Q&A]
I would like to use Emojis in command names, but then the searching is messed up.
![image](https://github.com/Textualize/textual/assets/49335821/dc0dff8c-f22d-41d1-85cb-9c37c644fdfe)

**Comment by davep** [comment]
*"the searching is messed up"* is very vague. Do you think you could detail what isn't working for you?

**Reply by Commandcracker** [reply]
I won't be able to find the command by typing dark or light, because it is not searching in the description.

**Reply by davep** [reply]
That would make sense: the `CommandPalette` is designed to search the commands, not the optional help text. That said, you're in charge of how the searching is done so there's nothing to stop you implementing a search approach that will work for your needs (although you'll struggle with the highlighting of hits).

Using emoji for *commands* that people are supposed to type doesn't seem like a great choice.
